README.md

```diff
@@ -1,5 +1,8 @@
 # k8s-authz
 [![Contributions Welcome](https://img.shields.io/badge/contributions-welcome-brightgreen.svg?style=flat)](https://github.com/casbin/k8s-authz/issues)
+[![Go Report Card](https://goreportcard.com/badge/github.com/casbin/k8s-authz)](https://goreportcard.com/report/github.com/casbin/k8s-authz)
+[![Coverage Status](https://coveralls.io/repos/github/casbin/k8s-authz/badge.svg?branch=master)](https://coveralls.io/github/casbin/k8s-authz?branch=master)
```

Comment:
Could you add this repo to coveralls? 

Also the goreport is showing grade B for my fork - https://goreportcard.com/report/github.com/ashish493/k8s-authz

It will get improved once the lints and tests are fixed.

Reply:
Coveralls enabled: https://coveralls.io/github/casbin/k8s-authz